internal/storage/raft/conformance_test.go

```diff
@@ -42,7 +42,8 @@ func TestBackend_Conformance(t *testing.T) {
 				_, follower := newRaftCluster(t)
 				return follower
 			},
-			SupportsStronglyConsistentList: true,
+			SupportsStronglyConsistentList:    true,
+			IgnoreWatchListSnapshotOperations: true,
```

Comment:
The snapshot is generated using a variably laggy follower in this half of the test, so there's no way to properly assert arrival order in the test.